desci-server/src/controllers/doi/mint.ts

```diff
@@ -0,0 +1,10 @@
+import { Request, Response, NextFunction } from 'express';
+
+import { BadRequestError, SuccessResponse, doiService } from '../../internal.js';
+
+export const mintDoi = async (req: Request, res: Response, next: NextFunction) => {
+  const { uuid } = req.params;
```

Comment:
generally i think uuid in current form is tough to work with (dot vs no dot, url-safe-base64 encoding not really standard) consider making the format explicit, or adding a comment about what format uuid is supposed to be, or use ensureUuidEndsWithDot function